tap_sparkthink/auth.py

```diff
@@ -1,47 +1,60 @@
-"""sparkthink Authentication."""
-
-
+"""gapi Authentication."""
```

Comment:
are we using GAPI auth, or is the code comment an artifact of where this code came from?